src/client/styles/components/FilterPopup.scss

```diff
@@ -382,3 +382,12 @@
     margin: 0 20px;
   }
 }
+
+// temporarily update styles for new DS Modal Component
+#chakra-modal--body-filters-button {
+  max-height: 500px;
```

Comment:
You will still have the same problem as before once the height reaches 500px -- the scrollbar will appear within the `Modal`. Try a larger value.

Reply:
I guess I'm more worried about Show Results disappearing than having users scroll? Whats the issue with the scroll bar?